Pod/Native/ParseState.swift

```diff
@@ -91,7 +90,7 @@ class ParseState {
                 
                 // The scenario description must be unique
                 let description = "\(description)_line\(example.lineNumber)"
-                scenarios.append(NativeScenario(description, steps: steps))
+                scenarios.append(NativeScenario(description, steps: steps, index: index + exampleIndex))
```

Comment:
This one is interesting - you're numbering each example step (which makes sense) but won't this cause issues with ordering i.e.

```
 Scenario: B This is a very simple example of ordering
        Given <name> should be executed before

    Scenario: A This is a very simple example of ordering too
        Given this should be executed after

        Examples:
        | name |
        | Alice |
        | Bob |
        | Eve |
```

Won't the indices look like?

A.Alice 0
A.Bob 1
A.Eve 2
B 1

I think you would have to account for the number of examples in a scenario when you increment the index between scenarios I.e B isn't A+1, it's A+3.